modules/mockk/src/commonTest/kotlin/io/mockk/it/CapturingTest.kt

```diff
@@ -85,8 +85,8 @@ class CapturingTest {
 
         assertFailsWith<MockKException> {
             verify {
-                mock.doSomething("1", capture(dataSlotId1))
-                mock.doSomething("2", capture(dataSlotId2))
```

Comment:
This would not fail if left as is, because with different values of the first argument only 1 call will be matched in each case